src/ig/IGExporter.ts

```diff
@@ -971,6 +971,19 @@ export class IGExporter {
       pathResourceDirectories = pathResources
         .map(directoryPath => path.join(this.inputPath, directoryPath))
         .filter(directoryPath => existsSync(directoryPath));
+      pathResources
+        .filter(directoryPath => directoryPath.endsWith(`${path.sep}*`))
```

Comment:
I'm wondering about how this works cross-platform (specifically on Windows). The [parameter docs](https://build.fhir.org/ig/FHIR/fhir-tools-ig/branches/master/CodeSystem-ig-parameters.html) say:
> Relative path to a location in the IG repository from which to load resources. If the path ends in /*, then that means all sub-folders

On Windows, however, I think `path.sep` will be `\`, right? (Unless running in Bash shell, which I suspect is what @mint-thompson might be doing).

I think the parameter value is probably supposed to be cross-platform and not OS-dependent, since you don't know what OS will build the IG. So... I think we probably need to update our handling of this to either:
* Always expect `/` path separators in this value, regardless of OS; OR
* Be agnostic about separators regardless of OS (supporting `/` and `\`)

I think it is probably OK to normalize on `/` (assuming a regression comes back clean) since the documentation says `/*`, which would imply normalization on `/`. But... I'm open to agnosticism too if that's what the team thinks is best.

Reply:
I can confirm that running the sample project on Windows when configured with `input/resources/nested/*` results in a warning for the nested resources. So, this will need to get updated as @cmoesel described.

Reply:
I think since the documentation says `/*`, let's just normalize on it. I ran a regression and things came back clean, but I tweaked the logic around paths after that so I'll double check with another regression. I updated the tests to use the normalized paths in the `path-resource` parameters that they set, so they should be covering the OS-agnostic paths in the CI. @mint-thompson - the windows tests pass, but do you mind testing this again to be sure?